modules/odc-importing-codebase-from-git-to-create-application.adoc

```diff
@@ -28,6 +28,7 @@ The following procedure walks you through the *From Git* option in the *Develope
 . After the Git URL is validated, the recommended builder image is selected and marked with a star. If the builder image is not auto-detected, select a builder image. For the `https://github.com/sclorg/nodejs-ex` Git URL, by default the Node.js builder image is selected.
 .. Optional: Use the *Builder Image Version* drop-down to specify a version.
 .. Optional: Use the *Edit import strategy* to select a different strategy.
+.. Optional: For Node builder image, use the **Run command** field to override the command to run the application.
```

Comment:
We call it "Node.js application" above, and based on their website this is how Node.js calls theirself.

```suggestion
.. Optional: For Node.js builder image, use the **Run command** field to override the command to run the application.
```